From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 30 Jun 2016 17:24:47 +0200
Subject: [PATCH] qtwebengine: add missing include

Without this building fails new newer glibc with:

.../touch_evdev_debug_buffer.h:37:15: error: field 'device_name_' has
incomplete type 'std::__cxx11::string {aka std::__cxx11::basic_string<char>}'

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 .../3rdparty/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h   | 1 +
 1 file changed, 1 insertion(+)

diff --git a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h
index 673a45467ade..c3187f6572eb 100644
--- a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h
+++ b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_evdev_debug_buffer.h
@@ -7,6 +7,7 @@
 
 #include <linux/input.h>
 #include <vector>
+#include <string>
 
 #include "base/memory/scoped_ptr.h"
 #include "ui/events/ozone/evdev/events_ozone_evdev_export.h"
